scapy/contrib/modbus.py

```diff
@@ -344,7 +344,7 @@ class ModbusWriteFileSubRequest(Packet):
     fields_desc = [ByteField("refType", 0x06),
                    ShortField("fileNumber", 0x0001),
                    ShortField("recordNumber", 0x0000),
-                   BitFieldLenField("recordLength", None, 16, length_of="recordData", adjust=lambda pkt, p: p/2),
+                   BitFieldLenField("recordLength", None, 16, length_of="recordData", adjust=lambda pkt, p: p//2),
```

Comment:
Doesn't this require `from __future__ import division` to be compatible with 2.7? I don't see it being imported.

Reply:
No it doesn't ^^ Python 2.7 has a native support of that

Reply:
Oh, right right. You need it `division` if you want `/` to be treated as float division.

Seems like a good step towards keeping things code base compatible with both would be to use `__future__` `absolute_import` `division` and `print_function` on all modules, right?

Reply:
And btw, thanks for helping push Python 3 compatibility for scapy 😃

Reply:
Again, in that case it's expected to be an int, not a float